Library/Homebrew/items.sh

```diff
@@ -27,7 +27,7 @@ homebrew-items() {
       -name spec -o \
       -name vendor \
       \) \
-      -prune -false -o -path "${find_include_filter}" |
+      -prune -false -o -path "${find_include_filter}" 2>/dev/null |
```

Comment:
Instead of hiding `stderr`: how about an early `return` if `${HOMEBREW_REPOSITORY}/Library/Taps` does not exist? Should be more performant that way. Note: probably worth adding a `# TODO` for that, too, that it should eventually use (e.g. shelling out to raw Ruby or something?) to extract these values from the JSON.

Reply:
Updated to handle this and adding initial attempt at getting `brew formulae` from JSON.